CONTRIBUTING.md

```diff
@@ -47,39 +47,51 @@ We follow the
 [Fork and pull model](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#fork-and-pull-model)
 in GitHub.
 
-You need to first fork this repository, and you can later on open a pull request
+You need to first fork this repository, and you can later open a pull request
 to propose changes from your own fork to the **master** branch in this source
 repository.
 
 GitHub provides detailed instructions in
-[Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo). In
-summary, you perform the follow steps to get your fork ready:
+[Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
 
-1.  Set up Git and authentication with GitHub.com.
+If you're doing this for the first time, the easiest way to get going is with the `gh` CLI tool:
 
-    https://docs.github.com/en/get-started/quickstart/set-up-git
+1. Install the `gh` CLI tool following the [instructions](https://github.com/cli/cli#installation)
 
-2.  Fork the `k8s-config-connector` repo. Instructions below assumes you also
-    name your fork as `k8s-config-connector`. If you use a different name for
-    the fork, you should replace the commands with the right name.
+1. Clone the repo.  We recommend you to create the local clone under the path
+    `~/go/src/github.com/GoogleCloudPlatform`. 
 
-    https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository
+    ```shell
+    gh repo clone github.com/GoogleCloudPlatform/k8s-config-connector ~/go/src/github.com/GoogleCloudPlatform/k8s-config-connector
+    cd ~/go/src/github.com/GoogleCloudPlatform/k8s-config-connector
+    ```
 
-3.  Clone your forked repo to your dev machine.
+1. Create a fork of the repo in your github account: `gh repo fork`
```

Comment:
Move this to a new line?

```
gh repo fork
```

Also I think we want to mention select "Yes" for "Would you like to add a remote for the fork?" This is what I see.

```
$ gh repo fork
✓ Created fork diviner524/k8s-config-connector
? Would you like to add a remote for the fork? Yes
✓ Added remote origin
```

Reply:
Added the `--remote` argument which does this automatically